### DISCUSSION TITLE
Help needed. TypeError: Cannot read properties of undefined (reading 'queryClient')

### DISCUSSION THREAD

**Discussion by El-Potato-Slayer** [wevm/wagmi#2466] [Q&A]
Hi there,

I've been using @wagmi/core and wagmi for about 2 months. I updated to the latest version of wagmi (1.0.8 for @wagmi/core and 1.0.9 for wagmi) and switched from WalletConnect to RainbowKit. Afterward, the following error popped up:
![image](https://github.com/wagmi-dev/wagmi/assets/43865875/133c0dc4-fa56-419b-b4b4-d67d40534e06)

I'm using the code from the RainbowKit [docs](https://www.rainbowkit.com/docs/installation#configure)

Does anyone have an idea as to what the issue what might be?

**Comment by tmm** [comment]
Did you wrap your app in a `<WagmiConfig>`? If you provide a minimal reproduction, someone might be able to help you out. Right now, it's difficult to tell what is happen.
